### PR TITLE
Updated Packages

### PR DESCRIPTION
**Pull request by imcodingideas**
Per issue #17 I went to reproduce this by running: npm run build. I was not able to reproduce this issue. 

Reading https://github.com/webpack/loader-utils/issues/56 I understood this to be an author problem. You may have had older packages installed. Hopefully this pull request solves it for you.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,21 +13,21 @@
...
-    "eslint-config-standard": "^10.2.0",
-    "eslint-plugin-react": "^6.10.3",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-node": "^4.2.2",
+    "eslint": "^4.3.0",
+    "eslint-loader": "^1.9.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-react": "^7.1.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-node": "^5.5.1",
```
Comment (sixlive) [inline package.json]:
```shell
detroitjs-com ▶︎ imcodingideas-build-warning ✓
⚡  npm i
npm ERR! code ETARGET
npm ERR! notarget No matching version found for eslint-plugin-node@^5.5.1
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'detroitjs.com'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tjmiller/.npm/_logs/2017-07-29T13_56_57_593Z-debug.log
```

With `npm install` I get the following error. With yarn it give me a prompt to select the correct dependency.

**Review by sixlive — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (imcodingideas) [reply package.json]: @sixlive My bad. I will fix this.

**Comment by imcodingideas** [comment]
I need to fix these conflicts I guess. 👍

**Comment by sixlive** [comment]
@imcodingideas I'm going to close this for now. If you want to re-open it with fixes I'd love to merge it!

**Pull request closed without merging**